src/App.js

```diff
@@ -1,25 +1,38 @@
+
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+
+const { ipcRenderer } = require('electron');
+
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {value: ''};
+  
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({value: event.target.value});
+  }
+  
+  handleSubmit(event) {
+    console.log(this.state.value);
+    ipcRenderer.send('print:value', this.state.value);
+    event.preventDefault();
+  }
+
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+        <form onSubmit={this.handleSubmit}>
+          <label>
+            File name:
+            <input type="text" value={this.state.value} onChange={this.handleChange} />
```

Comment:
Is input should be part of a label?

```suggestion
<form onSubmit={this.handleSubmit}>
  <label htmlFor="fileName">File name</label>
  <input type="text" name="fileName" value={this.state.value} onChange={this.handleChange} />
  <input type="submit" value="Submit" />
</form>
```